apps/gitness/src/layouts/PipelineStudioLayout.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react'
+import { Outlet } from 'react-router-dom'
+import Header from '../components/Header'
+
+const RepoLayout: React.FC = () => {
```

Comment:
is this compatible with the SandboxLayout?

Reply:
obsolete code